src/shared/ui/button/Button.tsx

```diff
@@ -0,0 +1,27 @@
+import styles from './button.module.css';
+import React from "react";
+
+type ButtonProps = React.ButtonHTMLAttributes<HTMLButtonElement> & {
+  size?: '' | 'semi' | 'large';
+  variant?: string;
+  onClick?: () => void;
+}
+
+export function Button(props: ButtonProps) {
+  const {
+    className,
```

Comment:
better to remove `className`
all styles should be controllable from Button component, not from outside

Reply:
I use it for adding global classes like margins, flex, paddings etc
Without declaration I cant put className in component